src/views/Scenario/ScenarioHook.js

```diff
@@ -17,7 +19,10 @@ export const useScenario = () => {
   const currentScenarioData = useCurrentScenarioData();
   const organizationId = useOrganizationId();
   const workspaceId = useWorkspaceId();
-
+  const datasets = useWorkspaceDatasets();
```

Comment:
`useWorkspaceDatasets` may not return all datasets, I think it's possible to have scenarios that use datasets that won't be in this list (either because the workspace filter has changed, or because the scenario has been created manually with the API). Maybe we should retrieve the dataset list from `useDatasets` instead?

Reply:
Good idea ! :+1: